What doesn't seem to work on my set up:

``` Bash
$ python tuf/ssl_crypto/tests/test_pycrypto_keys.py
Traceback (most recent call last):
  File "tuf/ssl_crypto/tests/test_pycrypto_keys.py", line 36, in <module>
    from ...ssl_commons import exceptions as ssl_commons_exceptions
ValueError: Attempted relative import in non-package
```

``` Bash
$ cd tuf/ssl_crypto/tests/
$ python test_pycrypto_keys.py
Traceback (most recent call last):
  File "test_pycrypto_keys.py", line 36, in <module>
    from ...ssl_commons import exceptions as ssl_commons_exceptions
ValueError: Attempted relative import in non-package
```

--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "toto/ssl_crypto"]
 	path = toto/ssl_crypto
-	url = git@github.com:/santiagotorres/ssl_crypto
+	url = https://github.com/santiagotorres/ssl_crypto

